examples/04.MuonScan/CosmicMuonsFromCircle.rml

```diff
@@ -18,19 +15,22 @@ First concept author G. Luzón (16-11-2020) -->
 
     <TRestGeant4Metadata name="restG4 run" title="MuonsFromCircle">
 
-        <parameter name="gdmlFile" value="setup.gdml"/>
+        <parameter name="gdmlFile" value="/tmp/tmp.SC7mdqBb3e/source/packages/restG4/examples/04.MuonScan/setup.gdml"/>
```

Comment:
This example will stop working if we add an absolute path like that isn't? I would better change directory to `/tmp/tmp.SC7mdqBb3e/source/packages/restG4/examples/04.MuonScan/` and launch the example.

Reply:
> This example will stop working if we add an absolute path like that isn't? I would better change directory to `/tmp/tmp.SC7mdqBb3e/source/packages/restG4/examples/04.MuonScan/` and launch the example.

My mistake, fixed it, it wasn't working on local.